src/style.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+}
+
+.search--bar {
+  width: 100%;
+  margin-top: 20px;
+}
+
+.bars--big {
+  height: 70px;
+}
+
+.bars--small {
+  height: 42px;
+}
```

Comment:
it is not a good idea to hardcode height. Use proper padding instead

Reply:
Use proper padding instead
?
how i can to do this

Reply:
check the padding at figma. 
And your fix will looks like (example, not real values)
In this case you will not hardcode `form` height. You'll use padding for input. And you don't need to use margin-left for an image
```
.input--small {
  padding: 20px;
}
```

Reply:
Still not fixed